homeassistant/components/kaleidescape/remote.py

```diff
@@ -0,0 +1,88 @@
+"""Kaleidescape remote."""
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+from kaleidescape import const as kaleidescape_const
+
+from homeassistant.components.remote import RemoteEntity
+
+from .const import DOMAIN as KALEIDESCAPE_DOMAIN, NAME as KALEIDESCAPE_NAME
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from typing import Any
+
+    from kaleidescape import Device as KaleidescapeDevice
+
+    from homeassistant.config_entries import ConfigEntry
+    from homeassistant.core import HomeAssistant
+    from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the platform from a config entry."""
+    entities = [KaleidescapeRemote(hass.data[KALEIDESCAPE_DOMAIN][entry.entry_id])]
+    async_add_entities(entities, True)
+
+
+class KaleidescapeRemote(RemoteEntity):
+    """Representation of a Kaleidescape device."""
+
+    def __init__(self, device: KaleidescapeDevice) -> None:
+        """Initialize remote."""
+        self._device = device
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID for device."""
+        return self._device.serial_number
+
+    @property
+    def name(self) -> str:
+        """Return the name of the device."""
+        return f"{KALEIDESCAPE_NAME} {self._device.system.friendly_name}"
+
+    @property
+    def is_on(self) -> bool:
+        """Return true if device is on."""
+        return self._device.power.state == kaleidescape_const.DEVICE_POWER_STATE_ON
```

Comment:
It doesn't look like this will ever change since there is nothing causing it to update and write state

Reply:
Remove remote platform for now.